usr/lib/cca_stdll/cca_specific.c

```diff
@@ -692,10 +698,8 @@ uint16_t cca_inttok_privkey_get_len(CK_BYTE * tok)
 /* Given a CCA internal token private key object, get the modulus */
 CK_RV cca_inttok_privkey_get_n(CK_BYTE * tok, CK_ULONG * n_len, CK_BYTE * n)
 {
-    uint16_t privkey_length, n_length;
-    uint32_t privkey_n_offset;
+    uint16_t n_length;
 
-    privkey_length = *(uint16_t *) & tok[CCA_RSA_INTTOK_PRIVKEY_LENGTH_OFFSET];
     n_length = *(uint16_t *) & tok[CCA_RSA_INTTOK_PRIVKEY_N_LENGTH_OFFSET];
```

Comment:
würg, can we rewrite this to maybe:
n_length = *((uint16_t *)(tok + CCA_RSA_INTTOK_PRIVKEY_N_LENGTH_OFFSET));
or at a minimum close the gap between "...& tok..." -> "...&tok..."
to make clear this is an address operator and not a binary and

Reply:
Yes, will remove the blank between & and tok.